<launch>
    <!-- 启动乌龟GUI节点 -->
    <node pkg="turtlesim" type="turtlesim_node" name="turtle1" output="screen"/>
    <!-- 键盘控制节点 -->
    <!-- 
        也可以不加这一行，启动后新建终端，运行
        rosrun turtlesim turtle_teleop_key
        来启动键盘
    -->
    <!-- 启动 turtle_move_server 节点 -->
    <node pkg="turtle_package" type="turtle_move_server" name="turtle_move_server" output="screen"/>
   
    <node pkg="turtle_package" type="turtle_Group4" name="Group4" output="screen"/>
    <node pkg="turtle_package" type="draw_helix" name="helix" output="screen"/>
    <node pkg="turtle_package" type="turtle_chase" name="turtle_chase" output="screen"/>

    <!--
    
    更换颜色
    <node pkg="turtle_package" type="change_color" name="change_color" output="screen"/>
    
    传入参数小海龟的名字
    <node pkg="turtle_package" type="pose_pub" name="pub1" args="Group1" output="screen"/>
    <node pkg="turtle_package" type="pose_pub" name="pub2" args="Group1_2" output="screen"/>
    
    <node pkg="turtle_package" type="vel_sub" name="control" output="screen"/>
    -->

    <!-- 启动 turtle_move_client 节点，并传递参数 
    <node pkg="turtle_package" type="turtle_chase" name="turtle_chase" output="screen">
        <param name="turtle_name" value="turtle1"/>
        <param name="x" value="5.0"/>
        <param name="y" value="5.0"/>
        <param name="is_straight" value="0"/>
    </node>-->
</launch>
